docs/source/fury-pybullet.rst

```diff
@@ -0,0 +1,606 @@
+Fury - pyBullet Integration Guide
```

Comment:
`.. _fury_pybullet:` is missing at the top of the file (before this title).

Also, FURY in capital everywhere